src/views/RecIM/views/Match/index.jsx

```diff
@@ -247,35 +297,12 @@ const Match = () => {
               setOpenMatchForm={(bool) => setOpenMatchForm(bool)}
               match={match}
             />
-            <GordonDialogBox
-              title="Admin Settings"
-              fullWidth
-              open={openSettings}
-              cancelButtonClicked={() => setOpenSettings(false)}
-              cancelButtonName="Close"
-            >
-              <br />
-              <Grid container alignItems="center" justifyContent="space-between">
-                <Grid item>
-                  <Typography>Permanently delete the match '{matchName}'</Typography>
-                </Grid>
-                <Grid item>
-                  <Button
-                    color="error"
-                    variant="contained"
-                    onClick={() => setOpenConfirmDelete(true)}
-                  >
-                    Delete this match
-                  </Button>
-                </Grid>
-              </Grid>
-            </GordonDialogBox>
             <GordonDialogBox
               title="Confirm Delete"
               open={openConfirmDelete}
               cancelButtonClicked={() => setOpenConfirmDelete(false)}
-              cancelButtonName="No, keep this team"
-              buttonName="Yes, delete this team"
+              cancelButtonName="No, keep this match"
+              buttonName="Yes, delete this match"
```

Comment:
I wonder if there is a way to make the button a bit smaller, the text does seem long but this is personal opinion